res/xml/devanagari.xml

```diff
@@ -0,0 +1,30 @@
+<?xml version="1.0" encoding="utf-8"?>
+<keyboard>
+  <row>
+    <key shift="0.35" width="0.9" key0="क" key1="ख" key2="घ" key3="ङ"  key4="ग"/>
+    <key              width="0.9" key0="च" key1="छ" key2="झ" key3="ञ"  key4="ज"/>
+    <key              width="0.9" key0="ट" key1="ठ" key2="ढ" key3="ण"  key4="ड"/>
+    <key              width="0.9" key0="त" key1="थ" key2="ध" key3="न"  key4="द"/>
+    <key              width="0.9" key0="प" key1="फ" key2="भ" key3="म"  key4="ब"/>
+    <key              width="0.9" key0="र" key1="ज्ञ" key2="ल" key3="य"  key4="व"/>
+    <key              width="0.9" key0="ह" key1="श" key2="ळ" key3="स"  key4="ष"/>
+  </row>
+  <row>
+    <key shift="0.35" width="0.9" key0="ा" key1="अ" key2="आ"/>
+    <key              width="0.9" key0="ि" key1="इ"/>
+    <key              width="0.9" key0="ी" key1="ई"/>
+    <key              width="0.9" key0="ु" key1="उ" key2="ऊ" key4="ू"/>
+    <key              width="0.9" key0="े" key1="ए" key2="ऋ" key4="ृ"/>
+    <key              width="0.9" key0="ै" key1="ऐ" key2="ऌ" key4="ॢ"/>
+    <key              width="0.9" key0="ो" key1="ओ" key2="औ" key4="ौ"/>
+  </row>
+  <row>
+    <key key0="्" key2="tab" key4="esc"/>
+    <key key0="ँ" key1="₹" key2="॑" key3="ॖ" key4="॓"/>
+    <key key0="ं" key1="ॐ" key2="ऽ" key3="ः" key4="&quot;"/>
+    <key key0="।" key1="," key2=";" key3="!" key4="\?"/>
+    <key key0="़" key1="॰" key2="०" key3="-" key4="॒"/>
+    <key key0="५" key1="१" key2="३" key3="७" key4="९" key5="४" key6="६" key7="२" key8="८"/>
```

Comment:
I wasn't really sure how to add the numbers, so I just threw them in a number pad